ompi/include/mpi.h.in

```diff
@@ -406,28 +404,15 @@ typedef int (MPI_Grequest_query_function)(void *, MPI_Status *);
 typedef int (MPI_Grequest_free_function)(void *);
 typedef int (MPI_Grequest_cancel_function)(void *, int);
 
-#if !OMPI_OMIT_MPI1_COMPAT_DECLS
 /*
- * Removed typedefs. These will be deleted in a future Open MPI release.
+ * Deprecated typedefs. These will be deleted in a future MPI standard.
```

Comment:
These *may* be deleted in a future version of the MPI standard.

"Deprecated", as defined by MPI, does *not* mean "will be deleted in a future version of the MPI standard."